diff --git a/index.d.ts b/index.d.ts
index 9ab61cdc3fbd9e8bdcded9aa8bbcf66e1818f155..c2b6821fec39c47bc96ad9219caac2ba44cd3bde 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -4,7 +4,7 @@ export interface FeedEntry {
   link?: string;
   title?: string;
   description?: string;
-  published?: Date;
+  published?: Date | string; // The string is not parsed into Date, but is confirmed to contain a string.
 }
 
 export interface FeedData {